platform/spire/src/infra.py

```diff
@@ -1,6 +1,7 @@
 import access
 import command
 import configuration
+import os
```

Comment:
Should adding this import go in the commit where the relevant functionality was added?

Reply:
This import should go in `spire: attempt multiple authentication methods` (but this would require a rebase).